### PR TITLE
feat: add fantom network

### PR DESCRIPTION
**Pull request by biendeveloper**
Fixes https://github.com/dimensiondev/maskbook/issues/4865

### CODE REVIEW

**Comment by UncleBill** [comment]
Hi, @biendeveloper , thanks for you work, but I'm sorry to tell you that Fantom integrating has been done in https://github.com/DimensionDev/Maskbook/pull/5036.

File: packages/dashboard/src/pages/Labs/components/SwapSettingDialog.tsx [hunk 1]
```diff
@@ -65,6 +71,12 @@ export default function SwapSettingDialog({ open, onClose }: SettingDialogProps)
             options: polygonOptions,
             onChange: (value: string) => Services.Settings.setPolygonNetworkTradeProvider(Number.parseInt(value, 10)),
         },
+        {
+            legend: t.labs_settings_swap_network({ network: 'Fantom' }),
+            value: useFantomNetworkTradeProvider(),
+            options: fantomOptions,
+            onChange: (value: any) => Services.Settings.setFantomNetworkTradeProvider(+value),
```
Comment (septs) [inline packages/dashboard/src/pages/Labs/components/SwapSettingDialog.tsx:78]:
```suggestion
            onChange: (value: string) => Services.Settings.setFantomNetworkTradeProvider(Number.parseInt(value, 10)),
```

Comment (septs) [reply packages/dashboard/src/pages/Labs/components/SwapSettingDialog.tsx]: @Jack-Works can simplify the code?

```suggestion
            onChange: Services.Settings.setFantomNetworkTradeProvider,
```

**Comment by biendeveloper** [comment]
> Hi, @biendeveloper , thanks for you work, but I'm sorry to tell you that Fantom integrating has been done in #5036.

Oh i didn't see that @UncleBill 😢

**Comment by UncleBill** [comment]
> > Hi, @biendeveloper , thanks for you work, but I'm sorry to tell you that Fantom integrating has been done in #5036.
> 
> Oh i didn't see that @UncleBill 😢

Sorry about that. I should mention in that issue early. I already remove the label "bounty: open" from it.

**Pull request closed without merging**